src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -9,20 +9,24 @@ metaDescription: Dbt Cloud with Airflow and Snowflake integration
 freshnessValidatedDate: never
 ---
 
-Our Dbt cloud integration with Airflow monitors the health of your dbt Cloud jobs
+Our dbt Cloud integration with Airflow monitors the health of your dbt Cloud jobs
```

Comment:
I'm OK with this change, but dbt Labs specifically spells dbt with a lowercase d. I only use an uppercase D when it starts a sentence. 

![image](https://github.com/newrelic/docs-website/assets/170976529/fa34a774-0937-46bd-93fc-a6d92f71b566)